sweep/sweeper.go

```diff
@@ -445,8 +429,19 @@ func (s *UtxoSweeper) feeRateForPreference(
 
 // collector is the sweeper main loop. It processes new inputs, spend
 // notifications and counts down to publication of the sweep tx.
-func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch,
-	bestHeight int32) error {
+func (s *UtxoSweeper) collector(blockEpochs <-chan *chainntnfs.BlockEpoch) {
+
```

Comment:
style nit: no newline needed when method signature is a single line

Reply:
Fixed.